### PR TITLE
WIP "AEROGEAR-2182 remove rhmap specific info from devel 1"

### PR DESCRIPTION
**Pull request by pmacko1**
https://issues.jboss.org/browse/AEROGEAR-2182

pr opened to enable commenting directly on text

### CODE REVIEW

**Comment by pmacko1** [comment]
verify this variable: "FH_CONFIG_CONTENT" Still used in one snippet

**Comment by matskiv** [comment]
LGTM  :+1: 
@pmacko1 what snippet are you talking about?

**Comment by pmacko1** [comment]
@matskiv ctrl-f this "        writeFile file: 'www/fhconfig.json', text: params.FH_CONFIG_CONTENT"
can't link, no changes there, jenkinsfile-create.asciidoc

**Comment by matskiv** [comment]
@pmacko1 Let's remove that line.

File: docs/digger/developer/jenkinsfile-create.asciidoc [hunk 1]
```diff
@@ -322,7 +290,6 @@ node("android"){
   ...
   stage("Prepare"){
       sh '. $NVM_DIR/nvm.sh && nvm install 6.9.5 && nvm alias default 6.9.5'
-      writeFile file: 'www/fhconfig.json', text: params.FH_CONFIG_CONTENT
```
Comment (pmacko1) [inline docs/digger/developer/jenkinsfile-create.asciidoc]: @matskiv

**Pull request merged**